src/attribute/attribute.controller.ts

```diff
@@ -1,31 +1,111 @@
-import { Controller, Get, Res, UseGuards } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Post, Put, Req, Res, UseGuards } from '@nestjs/common';
 import { AttributeService } from './attribute.service';
 import { ResponseService } from 'utils/response/customResponse';
 import { AuthGuard } from '@nestjs/passport';
+import { CreateAddressDto } from 'src/user/dto/create-address.dto';
+import { CreateAttributeDto } from './dto/attribute.dto';
+import { IdValidationPipe } from 'utils/validation/paramsValidation';
 
 @Controller('attribute')
 export class AttributeController {
   constructor(
     private readonly attributeService: AttributeService,
     private readonly responseService: ResponseService, // Inject the ResponseService
-  ) {}
+  ) { }
+
+  @Post()
+  @UseGuards(AuthGuard('jwt'))
+  async createAttribute(
+    @Body() createAttributeDto: CreateAttributeDto,
+    @Req() req,
+    @Res() res) {
+    try {
+      this.attributeService.create(createAttributeDto);
```

Comment:
this is not being handled by "await" please check and verify whether attributes are being created properly